packages/ui/react-ui-table/src/components/Table/Table.tsx

```diff
@@ -181,6 +184,10 @@ const TableImpl = <TData extends RowData>(props: TableProps<TData>) => {
     );
   }
 
+  if (process.env.NODE_ENV === 'development' && !getScrollElement) {
+    console.warn('Table: getScrollElement is not set. This is required for virtualized tables.');
```

Comment:
can you use @dxos/log package? there's a pending PR which would make this a lint error https://github.com/dxos/dxos/pull/5873

Reply:
Sure thing! I was wondering if there were a better way.